crates/sparrow-compiler/src/functions/collection.rs

```diff
@@ -0,0 +1,9 @@
+use sparrow_plan::InstOp;
+
+use crate::functions::{Implementation, Registry};
+
+pub(super) fn register(registry: &mut Registry) {
+    registry
+        .register("get<K: key, V: any>(map: map<K, V>, key: K) -> V")
+        .with_implementation(Implementation::Instruction(InstOp::Get));
```

Comment:
I expect I'll rename `InstOp::Get` to `InstOp::GetFromMap` internally, to then support `GetFromList` as well, but can leave for now.

Reply:
Also - prefer `Input.key | get(Input.map)` or `Input.map | get(Input.key)`? Currently is former, but latter looks more correct with pipe notation